prep-javascript-objects/main.js

```diff
@@ -18,4 +16,12 @@ const previousJob =
   " The person's previous job is: " + person['previousJob'] + '.';
 console.log(previousJob);
 
-console.log(person);
+const completePerson = {
+  firstName: 'Eduardo',
+  lastName: 'Gallegos',
+  job: 'Electrician',
+  previousJob: 'Door Dasher',
+  hobby: 'Fishing',
+};
```

Comment:
You are not supposed to be creating a brand new object. You are supposed to be creating the `job` & `previousJob` properties for your `person` object. The key thing here though, is that you are supposed to be creating these properties outside of the object literal, utilizing bracket notation and dot notation to create the properties.